Authenticator/Classes/OTPTokenListViewController.swift

```diff
@@ -167,12 +170,12 @@ extension OTPTokenListViewController /* UITableViewDataSource */ {
     }
 
     override func tableView(tableView: UITableView, commitEditingStyle editingStyle: UITableViewCellEditingStyle, forRowAtIndexPath indexPath: NSIndexPath) {
-        if (editingStyle == .Delete) {
+        if editingStyle == .Delete {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 157 characters